modules/rest-api/pages/rest-establish-credentials.adoc

```diff
@@ -63,6 +63,6 @@ curl -X POST http://<ip-address-or-domain-name>:8091/settings/web \
 
 Couchbase Server roles are described in xref:learn:security/roles.adoc[Roles].
 
-For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
+For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].
```

Comment:
`For each of the specific steps required in the provisioning process, see xref:rest-api:rest-configure-memory.adoc[Configuring Memory], xref:rest-api:rest-name-node.adoc[Naming a Node], xref:rest-name-cluster.adoc[Naming a Cluster], xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Single Node], and xref:rest-api:rest-establish-credentials.adoc[Establishing Credentials].`